tests/unit/client.py

```diff
@@ -236,6 +240,21 @@ def populate_server():
                 )
 
 
+# Avoid race condition while creating, deleting networks
+def avoid_network_race_condition():
+    import sqlite3
+    global db_dir  # Value of db_dir is populated by `create_setup`.
+    db_file = db_dir+'/haas.db'
+    conn = sqlite3.connect(db_file)
+    cur = conn.cursor()
+    while True:
+        que = len(cur.execute('select * from networking_action;').fetchall())
```

Comment:
You could just do `SELECT count(*) ...` and omit the call to `len`. With SQLAlchemy there's a `count` method that we use elsewhere in the source.